src/containers/other/NFTView/elements/Actions/Actions.tsx

```diff
@@ -165,7 +223,7 @@ const Actions = ({
 	 */
 	const ordered = isOwnedByUser
 		? [
-				actions[ACTION_TYPES.PlaceBid],
```

Comment:
Could reduce complexity by just having a `Bid` action type, and the `buttonComponent` in its `ActionBlock` changes based on ownership. Would require one less `ActionBlock` object above.